websubhub-examples/kafka-hub/README.md

```diff
@@ -83,6 +83,17 @@ Then to run the project execute the below command.
 bal run target/bin/kafka_hub_service.jar
 ```
 
+### Starting Multiple Hub Instances
+
+* Copy the `kafka_hub_service.jar` to another location.
+
+* Add another `Config.toml` which you could find in project to that location (Change `SERVER_ID` configuration in `Config.toml`).
```

Comment:
* Copy `Config.toml` which you could find the project root directory. 
* Change the `SERVER_ID` and `HUB_PORT `